core/common/assets/js/api/core/routes.js

```diff
@@ -61,15 +61,25 @@ export default class Routes extends Commands {
 		return true;
 	}
 
-	beforeRun( route, args ) {
-		if ( ! super.beforeRun( route, args ) ) {
+	validateRun( route, args = {} ) {
```

Comment:
`validateRun` calls `beforeRun` so, why need to change it?

Reply:
validateRun does validation only and beforeRun do anything else then validation.